remappers/hostmetrics/cpu.go

```diff
@@ -132,6 +141,38 @@ func addCPUMetrics(metrics pmetric.MetricSlice, resource pcommon.Resource, datas
 			timestamp:   timestamp,
 			doubleValue: &softirqPercent,
 		},
+	)
+
+	// TODO (lahsivjar): Remove the dependency on the system.cpu.logical.count
```

Comment:
Are we gaining anything if we do this ?
Since system.cpu.utilization is also an optional metric ?

Reply:
The code would have more complexity and branches when we depend on 2 metrics vs 1 metric.

Reply:
I have removed the TODO and [created an issue](https://github.com/elastic/opentelemetry-lib/issues/7) to discuss this so that the current PR can be reviewed quickly. Resolving this.